### DISCUSSION TITLE
How do I fix, "Couldn't connect to server"

### DISCUSSION THREAD

**Discussion by dhurbhat** [supabase/supabase#14756] [Questions]
I have a supabase edge function which is running locally, and I'm invoking this function in a pg_cron job using `pg_net`.  All of the responses to `http://localhost:54321/...` fail with `Couldn't connect to server`.
When I try your sample postman echo (`https://github.com/orgs/supabase/discussions/14747`), it returns a successful JSON payload as expected.

Can I not test pg_net locally?  
Does the `url` have to be a remote URL?

PS: I've also posted this as https://github.com/orgs/supabase/discussions/14747

**Comment by llanas** [comment]
You can check this related post https://github.com/supabase/pg_net/issues/79#issuecomment-1403145288
